web_widget_pattern/README.rst

```diff
@@ -0,0 +1,62 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+==============
+Input patterns
+==============
+
+This module was written to allow you to define a regex on character fields a user's input is tested against.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. depend on this module
+#. define a key ``pattern`` in the field's options dictionary::
+
+       <field name="email" options="{'pattern': '[\S]+@[\S]+'}" />
```

Comment:
Specify that the pattern is expressed as a regular expression and a general link to an explanation to this would be appreciated, like https://en.wikipedia.org/wiki/Regular_expression